modules/creating-serverless-apps-yaml.adoc

```diff
@@ -15,13 +15,13 @@ You can create a YAML file by copying the following example:
 apiVersion: serving.knative.dev/v1
 kind: Service
 metadata:
-  name: helloworld-go
+  name: hello-openshift
   namespace: default
 spec:
   template:
     spec:
       containers:
-        - image: gcr.io/knative-samples/helloworld-go
+        - image: docker.io/openshift/hello-openshift
           env:
             - name: TARGET
```

Comment:
same here, we should use RESPONSE, instead of TARGET for the hello-openshift image